ui/tutorial-renderer/build.gradle

```diff
@@ -0,0 +1,36 @@
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-kapt'
```

Comment:
I don't think you will need `kotlin-kapt` for this module since you aren't doing any annotation processing

Reply:
I needed it for the dataBinding